File: 01.JS-Basic/По сложни проверки/FruitShop.js
function fruitShop(input) {

    let fruit = input[0]
    let day = input[1]          //(banana / apple / orange / grapefruit / kiwi / pineapple / grapes), 
    let quantity = Number(input[2])   //(Monday / Tuesday / Wednesday / Thursday / Friday / Saturday / Sunday

    if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday") {
        switch (fruit) {
            case "banana": console.log((quantity * 2.5).toFixed(2)); break;
            case "apple": console.log((quantity * 1.2).toFixed(2)); break;
            case "orange": console.log((quantity * 0.85).toFixed(2)); break;
            case "grapefruit": console.log((quantity * 1.45).toFixed(2)); break;
            case "kiwi": console.log((quantity * 2.7).toFixed(2)); break;
            case "pineapple": console.log((quantity * 5.5).toFixed(2)); break;
            case "grapes": console.log((quantity * 3.85).toFixed(2)); break;
            default: console.log("error"); break;
        }
    }
    else if (day == "Saturday" || day == "Sunday") {
        switch (fruit) {
            case "banana": console.log((quantity * 2.7).toFixed(2)); break;
            case "apple": console.log((quantity * 1.25).toFixed(2)); break;
            case "orange": console.log((quantity * 0.9).toFixed(2)); break;
            case "grapefruit": console.log((quantity * 1.60).toFixed(2)); break;
            case "kiwi": console.log((quantity * 3).toFixed(2)); break;
            case "pineapple": console.log((quantity * 5.6).toFixed(2)); break;
            case "grapes": console.log((quantity * 4.2).toFixed(2)); break;
            default: console.log("error"); break;
        }
    }
    else { console.log("error") }

}
fruitShop(["kiwi",
"Monday",
"2.5"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatementsAdvanced/FruitShop.java
package ConditionalStatementsAdvanced;

import java.util.Scanner;

public class FruitShop {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String fruit = scan.nextLine();
        String day = scan.nextLine();
        double quantity = Double.parseDouble(scan.nextLine());

        switch (day) {
            case "Monday":
            case "Tuesday":
            case "Wednesday":
            case "Thursday":
            case "Friday":
                switch (fruit) {
                    case "banana":
                        System.out.printf("%.2f", quantity * 2.5);
                        break;
                    case "apple":
                        System.out.printf("%.2f",quantity * 1.2);
                        break;
                    case "orange":
                        System.out.printf("%.2f",quantity * 0.85);
                        break;
                    case "grapefruit":
                        System.out.printf("%.2f",quantity * 1.45);
                        break;
                    case "kiwi":
                        System.out.printf("%.2f",quantity * 2.7);
                        break;
                    case "pineapple":
                        System.out.printf("%.2f",quantity * 5.50);
                        break;
                    case "grapes":
                        System.out.printf("%.2f",quantity * 3.85);
                        break;
                    default:
                        System.out.printf("error");
                }
                break;
            case "Saturday":
            case "Sunday":
                switch (fruit) {
                    case "banana":
                        System.out.printf("%.2f",quantity * 2.7);
                        break;
                    case "apple":
                        System.out.printf("%.2f",quantity * 1.25);
                        break;
                    case "orange":
                        System.out.printf("%.2f",quantity * 0.90);
                        break;
                    case "grapefruit":
                        System.out.printf("%.2f",quantity * 1.60);
                        break;
                    case "kiwi":
                        System.out.printf("%.2f",quantity * 3.00);
                        break;
                    case "pineapple":
                        System.out.printf("%.2f",quantity * 5.60);
                        break;
                    case "grapes":
                        System.out.printf("%.2f",quantity * 4.20);
                        break;
                    default:
                        System.out.println("error");
                }
                break;
            default:
                System.out.println("error");
                break;
        }
    }
}
